front/lib/vaults.ts

```diff
@@ -17,3 +17,22 @@ export function getVaultIcon(
 export const getVaultName = (vault: VaultType) => {
   return vault.kind === "global" ? "Company Data" : vault.name;
 };
+
+import type { DataSourceViewCategory } from "@dust-tt/types";
+
+import { isFolder, isWebsite } from "@app/lib/data_sources";
+import type { DataSourceResource } from "@app/lib/resources/data_source_resource";
+
+export const getDataSourceCategory = (
```

Comment:
Impressive you managed to write twice the same function in the same PR for a function that already existed 😅 🤯 !